src/main/java/chzzk/grassdiary/domain/reward/RewardType.java

```diff
@@ -0,0 +1,19 @@
+package chzzk.grassdiary.domain.reward;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class RewardType {
+    @Id
+    @GeneratedValue( strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String type;
+}
```

Comment:
```suggestion
public enum RewartType {
    PLUS_POINT("일기 작성"),
    PURCHASE_ITEM("상점 구매");

    private final String displayName;

    RewartType(String displayName) {
        this.displayName = displayName;
    }
}
```

 ### RewardType을 Enum으로 변경
 수령 내역의 타입을 알기만 하면 되고, 해당 타입에 대한 추가 정보나 동작이 필요하지 않다.

Reply:
해당 클래스의 이름이 `RewartType`라고 살짝 오타가 난 것 같은데, 해당 부분만 수정해주시면 좋을 것 같아요 👍